azkaban-common/src/main/java/azkaban/constants/ServerInternals.java

```diff
@@ -26,5 +26,12 @@
   public static final String AZKABAN_EXECUTOR_PORT_FILENAME = "executor.port";
 
   public static final String AZKABAN_SERVLET_CONTEXT_KEY = "azkaban_app";
-  
+
+
+  // Memory check retry interval when OOM in ms
+  public static final long MEMORY_CHECK_INTERVAL = 1000*60*10;
```

Comment:
i think 1 min is too short, maybe 5 mins. Memory issue should't go away so soon.